src/prog_algs/metrics/toe_profile_metrics.py

```diff
@@ -31,13 +31,13 @@ def alpha_lambda(toe_profile : ToEPredictionProfile, ground_truth : dict, lambda
             # If keys not provided, use all
             keys = params.setdefault('keys', toe.keys())
 
+            bounds = {key : [ground_truth[key] - alpha*(ground_truth[key]-t_prediction), ground_truth[key] + alpha*(ground_truth[key]-t_prediction)] for key in keys}
```

Comment:
```suggestion
            bounds = {key : [gt - alpha*(gt-t_prediction), gt + alpha*(gt-t_prediction)] for key, gt in itemize(keys)}
```

Your solution is fine, I just think this is a bit cleaner